### DISCUSSION TITLE
[FEATURE-REQUEST] Cuando se quiere agregar un cambio de estado en una SM, puede quedar por defecto "Ninguno", en vez de "Manual"?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#283] [3. Ideas & Feature Requests] [answered]
Hola

Cuando uno está trabajando en una SM, ya sea nueva o existente, y se quiere agregar varios estados, para luego configurarlos y a nuevos cambios estados y éstos arrojan "Manual".
Es un detalle muy menor, pero a veces incomoda cuando estás implementando, teniendo los survey triggers.

Adjunto una imagen de un ejemplo

![image](https://user-images.githubusercontent.com/61419471/122946025-041bc680-d347-11eb-9a4b-dc790aa23116.png)

Gracias :>

**Comment by guillevalin** [comment, marked as answer]
@NicolasPLG Es posible, lo pensamos porque creemos que es más intuitivo que cuando creas una máquina de estado, enseguida aparezcan las "transiciones" de ellas en el botón de actions, que es la nueva forma de usar tasks.

El "None" tiene sentido como bien dices si quieres usar rutinas asociadas a triggers, pero es un patrón de modelado que servía antes de la existencia de required surveys y de la unificación de channel con tasks.
